integration-tests/contract-permits-set.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { CONFIGS } from "./config";
+import { MichelsonMap, MichelCodecPacker } from "@taquito/taquito";
+import { permit_admin_42_set} from "./data/permit_admin_42_set";
+import { Parser } from '@taquito/michel-codec'
+import { importKey } from '@taquito/signer';
+import { buf2hex, hex2buf } from "@taquito/utils";
+
+const blake = require('blakejs');
+
+CONFIGS().forEach(({ lib, rpc, setup }) => {
+  const Tezos = lib; 
+  Tezos.setPackerProvider(new MichelCodecPacker());
+  const bob_address = 'tz1Xk7HkSwHv6dTEgR7E2WC2yFj4cyyuj2Gh';
+ 
+  const errors_to_missigned_bytes = (errors: any[]) => {
+    const errors_with = errors.map(x => x.with).filter(x => x !== undefined);
+    if (errors_with.length != 1){
+      throw ['errors_to_missigned_bytes: expected one error to fail "with" michelson, but found:', errors_with]
+    } else {
+      const error_with = errors_with[0];
+      const p = new Parser();
+      const michelsonCode = p.parseJSON(error_with);
```

Comment:
lines 21 and 22 are not used. Maybe consider removing these lines.

Reply:
Removed. I had left them in as they were in the Michael Klien example. But yeah not used ..